### DISCUSSION TITLE
How to swtich to light to dark theme

### DISCUSSION THREAD

**Discussion by crisfcodes** [react-native-elements/react-native-elements#3388] [Q&A] [answered]
I've read the documentation of react native elements, but it is not clear to me how to update the theme from light to dark, I see that you can use `updateTheme` and change `mode` them property, but I don't know if it is the right way to do it.

**Comment by arpitBhalla** [comment, marked as answer]
You may refer this
https://github.com/react-native-elements/react-native-elements/blob/e1790719d8b7fc8b6f70e8ac806a1a6d8bf8f64a/example/src/navigation/DrawerNavigator.tsx#L68-L72

**Reply by myhendry** [reply]
I am able to toggle the theme that sets my button to a different color depending on light or dark mode

However, how do i toggle the View component from react-native?

**Reply by arpitBhalla** [reply]
You mean the background?

**Comment by myhendry** [comment]
yup, the background
